framework/Jenkinsfile

```diff
@@ -59,20 +59,15 @@ def teardown_mocha(test_name) {
 			dir('framework') {
 				sh """
 				rm -rf coverage_mocha_${test_name}; mkdir -p coverage_mocha_${test_name}
-				npx istanbul report --root framework/test/mocha/.coverage-unit/ --dir framework/test/mocha/.coverage-unit/
-				cp framework/test/mocha/.coverage-unit/lcov.info coverage_mocha_${test_name}/ || true
-				npx istanbul report cobertura --root framework/test/mocha/.coverage-unit/ --dir framework/test/mocha/.coverage-unit/
-				cp framework/test/mocha/.coverage-unit/cobertura-coverage.xml coverage_mocha_${test_name}/ || true
-				curl http://localhost:4000/coverage/download --output functional-coverage.zip
-				unzip functional-coverage.zip lcov.info -d coverage_mocha_${test_name}/functional/
+				cp test/mocha/.coverage-${test_name}/* coverage_mocha_${test_name}/ || true
```

Comment:
Actually having test_name variable here is redundant. Because all mocha tests (unit, integration, functional) are creating their coverage reports to `mocha/.coverage-unit` folder. But I confirmed with @Nazgolze, since all parallel tests are executed in different node it's not a problem. Technically, two test suites can be executed in the same node as well but not at the same time and `cleanWs` function in Jenkinsfile, ensures workspace is cleaned before.

I'd suggest to hard code unit here. I'm gonna create an issue to use different folders for different test types when running mocha tests and reference it to this discussion.

Reply:
After having the discussion, since we don't need coverage reports other than integration and deprecating mocha gradually, we can ignore this comment.